packages/actionmenu/src/react/item.tsx

```diff
@@ -43,99 +47,107 @@ const styles = {
   textOnly: () => css(stylesheet['.psds-actionmenu__text-only'])
 }
 
-interface Props extends Omit<HTMLAttributes<HTMLLIElement>, 'onClick'> {
-  disabled?: boolean
-  tagName?: string
+interface ItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'onClick'> {
   className?: string
-  origin?: string
-  onClick?: (event: React.MouseEvent, value: string | number) => void
+  disabled?: boolean
+  href?: string
   nested?: React.ReactNode
+  onClick?: (event: React.MouseEvent, value: string | number) => void
+  origin?: string
+  tagName?: string
   value?: string | number
 }
-
-export const Item = forwardRef<HTMLLIElement, Props>(
-  (
-    {
+export const Item = forwardRef<HTMLLIElement, ItemProps>(
+  (props, forwardedRef) => {
+    const {
+      children,
+      className,
       disabled,
       nested,
+      onClick,
       origin,
+      tagName: TagName = tagName.a,
```

Comment:
the react team has said they are going to get ride of defaultProps. i moved the default into this destructure statement. idk if this is something we should do everywhere...

Reply:
First I've heard of that going away.  Do you have a url where I can confirm and hear what their reasoning is?

If it is going away, I'm all for replacing React.defaultProps with an alternative default strategy.

Reply:
https://github.com/facebook/react/pull/16210